sdk/digitaltwins/digitaltwins/README.md

```diff
@@ -0,0 +1,123 @@
+# Azure IoT Digital Twins client library for Javascript
+
+Azure Digital Twins is a developer platform for next-generation IoT solutions that lets you create, run, and manage digital representations of your business environment, securely and efficiently in the cloud. With Azure Digital Twins, creating live operational state representations is quick and cost-effective, and digital representations stay current with real-time data from IoT and other data sources. If you are new to Azure Digital Twins and would like to learn more about the platform, please make sure you check out the Azure Digital Twins [official documentation page](https://docs.microsoft.com/azure/digital-twins/overview).
+
+For an introduction on how to program against the Azure Digital Twins service, visit the [coding tutorial page](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-code) for an easy step-by-step guide. Visit [this tutorial](https://docs.microsoft.com/azure/digital-twins/tutorial-command-line-app) to learn how to interact with an Azure Digital Twin instance using a command-line client application. Finally, for a quick guide on how to build an end-to-end Azure Digital Twins solution that is driven by live data from your environment, make sure you check out [this helpful guide](https://docs.microsoft.com/azure/digital-twins/tutorial-end-to-end).
+
+This library provides access to the Azure Digital Twins service for managing twins, models, relationships, etc.
+
+Use the client library to:
+
+- Create and manage models and components.
+- Create and manage digital twins.
+- Create and manage event routes.
+- Create and manage relationships between the models and twins.
+- Publis telemetry to digital twin or component.
```

Comment:
Publish

Reply:
Fixed

---
In reply to: [455390509](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r455390509) [](ancestors = 455390509)